packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -74,6 +80,7 @@ function MaybeIframe( { children, contentRef, shouldIframe, styles, style } ) {
 			name="editor-canvas"
 		>
 			{ children }
+			<SVG dangerouslySetInnerHTML={ { __html: filters } } />
```

Comment:
This feel bad. I think the alternative is to fetch the filters from global styles like we do in the Site Editor.